### ISSUE TITLE
The keyboard is automatically deactivated after device reboot

### ISSUE THREAD

**Issue by jerouy** [PhilippC/keepass2android#206]
Hi,

I noticed this keepass2android keyboard would be automatically after device reboot. I have to activate the keyboard manually upon every reboot.

Is there something you can look into?

I am using Android 7.0 and keepass2android offline.

Thanks!

**Comment by mnl1121** [comment]
I also have this issue as well. I'm running stock Android 7.0 on a Galaxy S7.

**Comment by PhilippC** [comment]
does your device switch to another input method but KP2A keyboard is still enabled? Or is it completely disabled in the settings?

**Comment by mnl1121** [comment]
No, KP2A keyboard gets completely disabled. I have to go back into the system settings and reenable. Mine doesn't get disabled every phone restart, I don't know when it happens as I use KP2A every a couple of days usually but by the time I go to use it again it's disabled.

**Comment by jerouy** [comment]
Thanks Philip, for looking into this.

The keyboard would be disabled completely. I have to go to Settings > Languages and input > Keyboard
to turn the keyboard back on.

I do notice though, this behaviour only happens on my nVidia SHIELD Tablet K1(Android 7.0). It does not happen on my LG(Google) Nexus 5(Android 6.01).

Hope that helps.

Thanks!

**Comment by rodmur** [comment]
I'm not sure if my problem is the same, my keyboard was showing up fine until, I believe, I upgraded to Android 8.1.  Now I can't access the keyboard, it isn't deactivated, it just doesn't want to appear.

**Comment by rodmur** [comment]
I'm using Keepass2Android version 1.03, I use a Nexus 6P running Android 8.1

**Comment by PhilippC** [comment]
@rodmur do you try to switch from GBoard to KP2A Keyboard? If yes, please see #56

**Comment by OldSunGuy** [comment]
I am seeing this issue without GBoard installed using Keepass2Android 1.05e running Android 7.0 on a Samsung Galaxy S6. It requires me to acknowledge that KeePass2Android may be able to collect all typed text.

**Comment by OldSunGuy** [comment]
I had the KP2A AutoFillPlugin installed. After removal, the KP2A keyboard remains enabled.
